src/components/routines.js

```diff
@@ -1,12 +1,37 @@
 import React, {useState, useEffect} from "react";
+import {fetchRoutines} from '../api';
+import {RoutineSingle} from 'RoutineSingle';
 const Routines = () => {
-
-
+    const [routines, setRoutines] = useState([])
+    const getRoutines = async () => {
+        setRoutines(await fetchRoutines())  
+    }
+    useEffect(() => {        
+        getRoutines()
+    }, [])
+    console.log(routines)
 return (
     <>
-    <h3>Routine:</h3>
-    <h3>Goal:</h3>
-    <h3>Creator:</h3>
+    {routines.map(routine => {
+
+        console.log(routine)
```

Comment:
when youre ready to push up your branch, remember to remove all the console.logs